### DISCUSSION TITLE
Hvordan kommer man væk fra oprettelsesbilledet?

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#15] [Questions] [answered]
Fra Emma:

> Jeg kan ikke længere logge ind på portalens forside uden at denne formel dukker op som default (se skærmbillede), når jeg forsøger at fortsætte og oprette profilen, for derefter at afvise den som spam i vores admin. backend, bliver den stadig ved med at være der. har forsøgt at slette browserhistorik, genstarte faner, computer osv. Det startede i går. <

![image](https://github.com/user-attachments/assets/6c7ca821-39cf-492e-ad9d-7fb2db3af467)

Jeg tror også selv, at jeg er stødt på problemet før. Måske er det noget, der opstår, når man har to brugere med forskellige e-mail adresser. I øvrigt er der ikke nogen måde at komme væk fra denne popup på. Man kan ikke f.eks. gå tilbage i browseren.

**Comment by antonlund18** [comment, marked as answer]
Det sker når du er logget ind i portalen, men ikke er blevet oprettet som frivillig hos organisationen endnu. Som workaround kan man gå ind i ens browser indstillinger, trykke på Applikationsfanen, trykke på 'Local Storage' og slette den token der ligger derinde:

![image](https://github.com/user-attachments/assets/2d21007e-9534-498a-9f69-3fc46e30b9af)

Det nemmeste ville være at tilføje en 'Log ud' knap under 'Fortsæt', men jeg er lidt bange for at det måske ville miste nogle nye frivillige. Har i nogle ønsker til hvordan i gerne ville have det opførte sig?

**Reply by MartinQwist** [reply]
Jeg forstår. Foreløbigt er det jo så ikke et problem, som ret mange vil opleve fordi man typisk kun opererer i en frivilligorganisation. Jeg vender den lige med mine kolleger og vender tilbage.
